Source/Fuse.Common/UpdateManager.uno

```diff
@@ -82,8 +81,18 @@ namespace Fuse
 		public List<UpdateListener> Onces = new List<UpdateListener>();
 		public List<UpdateListener> OncesPending = new List<UpdateListener>();
 		
-		public List<UpdateAction> PhaseDeferredActions = new List<UpdateAction>();
-		public int PhaseDeferredActionsAt = -1;
+		public Dictionary<int, Queue<UpdateAction>> PhaseDeferredActions = new Dictionary<int, Queue<UpdateAction>>();
```

Comment:
It would actually make it faster. YOu don't need to compare priroity at all, you could simply take the first one. Indeed you could drop the `Dictionary` and just use a fixed array, which should be faster than iteration each time.